### PR TITLE
Network DB access

### PR DESCRIPTION
**Pull request by ScottyPoi**
This updates the way that `network` classes (`HistoryNetwork`, `StateNetwork`, etc) interact with the database.

These network classes now have direct access to the `DBManager`, and all of the `put`, `get`, `del`, and other database methods.

This does away with some of the `bind` usage.  Network database methods and defined like other async methods.  This removes the extraneous `networkId` parameter from `put`, `get`, and `prune`.

--
These changes will make way for greater database and `node radius` management on the sub-network level.

### CODE REVIEW

**Comment by ScottyPoi** [comment]
@acolytec3 

bye bye `bind`  :stuck_out_tongue_winking_eye:

**Comment by acolytec3** [comment]
Your efforts to remove hard to parse code are commendable.

**Review by acolytec3 — APPROVED** [review]
